### PR TITLE
Minor improvements to glossary

### PR DESCRIPTION
**Pull request by avivace**

### CODE REVIEW

**Comment by brian-brazil** [comment]
Can you add the DCO?

File: content/docs/introduction/glossary.md [hunk 1]
```diff
@@ -35,17 +35,16 @@ a single metric if it is part of direct instrumentation, or many metrics if it i
...
 ### Endpoint
 
 A source of metrics that can be scraped, usually corresponding to a single process.
 
 ### Exporter
 
-An exporter is a binary that exposes Prometheus metrics, commonly by converting
-metrics that are exposed in a non-Prometheus format into a format Prometheus supports.
+An exporter is a binary running alongside the application you
+want to obtain metrics from, that exposes Prometheus metrics, commonly by converting metrics that are exposed in a non-Prometheus format into a format Prometheus supports.
```
Comment (beorn7) [inline content/docs/introduction/glossary.md]: This is getting a bit hard to parse. I'd split it into two sentences:
```
An exporter is a binary running alongside the application you
want to obtain metrics from. The exporter exposes Prometheus metrics, commonly by converting metrics that are exposed in a non-Prometheus format into a format that Prometheus supports.
```

Comment (avivace) [reply content/docs/introduction/glossary.md]: Looks good to me

Comment (beorn7) [reply content/docs/introduction/glossary.md]: @avivace if you could update your PR with the suggested change, we could merge it.

Comment (avivace) [reply content/docs/introduction/glossary.md]: @beorn7 Yup, sorry, got busy lately. Will work on it as soon as I have some time

**Comment by avivace** [comment]
@beorn7 Should be okay now

**Comment by brian-brazil** [comment]
Thanks!
